platform/consul/monitor.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package consul
+
+import (
+	"reflect"
+	"sort"
+	"time"
+
+	"github.com/golang/glog"
+
```

Comment:
The typical pattern is to group golang standard deps first, then third party packages, then istio packages. That's not enforced, but looks nicer.